src/modifierSummoningSickness.ts

```diff
@@ -42,7 +42,7 @@ export default function registerSummoningSickness() {
             // Skip turn
             return true;
         },
-        onTurnEnd: async (unit: Unit.IUnit, underworld: Underworld) => {
+        onTurnEnd: async (unit: Unit.IUnit, prediction: Boolean, underworld: Underworld) => {
```

Comment:
Boolean should be lowercase.  In js Boolean and boolean are actually different and so is String and string (always prefer the lower case).  It's weird I know... lol